src/js/common/schemaform/save-load-actions.js

```diff
@@ -100,6 +100,7 @@ function migrateFormData(savedData, savedVersion, migrations) {
  */
 export function saveInProgressForm(formId, version, returnUrl, formData) {
   // Double stringify because of api reasons. Olive Branch issues, methinks.
+  // TODO: Stop double stringifying
```

Comment:
Re: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3046#issuecomment-305872275
Just thought I'd record this while I was thinking about it. :man_shrugging: